blog.html

```diff
@@ -9,47 +9,88 @@
   <body class="blog">
     <div class="blog-container">
       <!-- Beginning of Nav -->
-      <div class="nav-blog">
+      <div class="header-container header-blog">
 
-        <div class="logo-top">
+        <div class="logo-top blog-logo">
           <a href="index.html">Relaxr</a>
         </div>
 
-        <div class="nav-right">
+        <div class="nav-container">
+          <div class="nav-item-blog">
           <a href="#">About</a>
-        </div>
+          </div>
 
-        <div class="nav-right">
-          <a href="#">FAQ</a>
-        </div>
+          <div class="nav-item-blog">
+            <a href="#">FAQ</a>
+          </div>
 
-        <div class="nav-right">
-          <a href="#">Team</a>
-        </div>
+          <div class="nav-item-blog">
+            <a href="#">Team</a>
+          </div>
 
-        <div class="nav-right">
-          <a href="#">Contact Us</a>
-        </div>
+          <div class="nav-item-blog">
+            <a href="#">Contact Us</a>
+          </div>
+
+          <div class="nav-item-blog">
+            <a href="./blog.html">Blog</a>
+          </div>
 
-        <div class="nav-right">
-          <a href="./blog.html">Blog</a>
         </div>
-      </div>
+
+    </div>
       <!-- End of Nav -->
 
       <!-- Beginning of main-column -->
```

Comment:
@PalomaCalles I would have tried to create another div here to wrap around the blog and the sidebar together. wrapping these items and setting in on display: flex would arranged the divs inside....just a thought for upcoming projects :)